src/Compiler/Directory.Packages.props

```diff
@@ -1,23 +0,0 @@
-﻿<Project>
-  <PropertyGroup>
-    <_RoslynPackageVersion>4.9.2</_RoslynPackageVersion>
-    <_RoslynPackageVersion Condition="'$(DotNetBuildSourceOnly)' == 'true'">$(MicrosoftCodeAnalysisCommonPackageVersion)</_RoslynPackageVersion>
-  </PropertyGroup>
-
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
-  <ItemGroup>
-
-    <!--
-      The compiler portion of razor is specifically pinned to an earlier version of the
-      roslyn compiler. That maximizes our compatibility with earlier versions of .NET SDK
-      inside of visual studio
-
-      https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle
-    -->
-
-    <PackageVersion Update="Microsoft.CodeAnalysis.Common" Version="$(_RoslynPackageVersion)" />
-    <PackageVersion Update="Microsoft.CodeAnalysis.CSharp" Version="$(_RoslynPackageVersion)" />
-    <PackageVersion Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(_RoslynPackageVersion)" />
-    <PackageVersion Update="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="$(_RoslynPackageVersion)" />
```

Comment:
Now that this is gone we should be able to remove most of the `VersionOverride=...` in `<PackageReference` under the compiler directory.